task_board/spec/factories/tasks.rb

```diff
@@ -0,0 +1,9 @@
+FactoryBot.define do
+  factory :task do
+    sequence(:name) { 'test1' }
+    sequence(:description) { 'test1_description' }
+    sequence(:end_date) { Time.now.next_week }
+    sequence(:priority) { :底 }
```

Comment:
`sequence` は連番等を使って値を少しずつ変化させる時に使うものなので、変化させる必要がない/してほしくないところには使わないようにしてください。
`sequence` で定義すると思わぬところが変化したりしてしまうと思います。
https://qiita.com/kuboon/items/8897aca59dc968ca6d3c

Reply:
また、enum の値には日本語を使わないようにしてください。
Railsは enum の値に対して自動でスコープやメソッドを生成するのですが、これが日本語になってしまうのでコーディング規約的に良くないです。

↓こんな感じになります
```ruby
irb(main):015:0> t = Task.new(priority: '底')
irb(main):016:0> t.底?
=> true
irb(main):017:0> t.高?
=> false

irb(main):019:0> Task.底.count
   (0.7ms)  SELECT COUNT(*) FROM `tasks` WHERE `tasks`.`priority` = 1
=> 1
```

Reply:
`next_week` メソッドは次の月曜日になるので、テスト実行タイミングによって残り日数が変わることを理解していないと正しいテストが書けなくなりそうです。
明示的に n日後 にしたほうがわかりやすそう。

```ruby
[3] pry(main)> 2.days.ago.next_week
Mon, 26 Oct 2020 00:00:00 JST +09:00
[4] pry(main)> 2.days.since.next_week
Mon, 26 Oct 2020 00:00:00 JST +09:00
```